deployment/kube/prod/frontend.yaml

```diff
@@ -39,6 +45,12 @@ spec:
         # starting the pod. This is useful when debugging, but should be turned
         # off in production.
         imagePullPolicy: Always
+        resources:                                                              
+         limits:                                                                
+           memory: 2396Mi                                                       
```

Comment:
Isn't this still a bit small compared to the 3.75 GB available?

Reply:
3.75 but 1G is reserved for kubelet and I've tested with 2.5G and I was starting to get some evicted due to not enough ram. We also have fluentd that uses 200M~ so it add up quickly.

2.4G is really the max we can get out of this instance.

Reply:
OK, that's unfortunate but thanks for the explanation.